lib/core/test/integration/Test/Integration/Framework/TestData.hs

```diff
@@ -245,6 +246,10 @@ errMsg403InvalidTransaction :: String
 errMsg403InvalidTransaction = "I can't process this payment because it contains at least\
     \ one payment output of value 0. This isn't supported by the current core nodes."
 
+errMsg403CannotEstimateFee :: String
+errMsg403CannotEstimateFee = "I can't estimate transaction fee because transaction contains\
+    \ at least one payment output of value 0."
```

Comment:
We're probably going to need two different messages here right, one for each case?

Reply:
Not sure how we can manage to test the other case though ....

Reply:
two messages here, one is already caught in http-bridge specific integration tests, the other will be caught on specific jormunagandr integration tests (and when the corresponding case will be added then the second value will be added)

Reply:
I unified `errMsg403InvalidTransaction` with `errMsg403CannotEstimateFee` as not coin selection validation is used (usisng specific implementation for both backends)